### DISCUSSION TITLE
What is the name of the farthest point on 🌍 from any landmass?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1123] [Galaxy Trivia] [answered]
**Bonus:** Pin 📌 the exact location on a map.

**Comment by tomerf** [comment, marked as answer]
The farthest point from any land is in the Pacific Ocean, popularly known as 'Point Nemo'. It is named after the famous submarine sailor from Jules Verne's Twenty Thousand Leagues Under the Sea.
It's located at [48°52.6′S 123°23.6′W](https://www.google.com/maps?hl=en&q=48%C2%B052.6%E2%80%B2S+123%C2%B023.6%E2%80%B2W), about 2,688 kilometers from the nearest land.

**Comment by peterbaccaro** [comment]
The most distant point from land is the **Pacific pole of inaccessibility** (also called **"Point Nemo"**, at a region known as the spacecraft cemetery), which lies in the South Pacific Ocean at 48°52.6′S 123°23.6′W, about 2,688 km (1,670 mi) from the nearest land.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/31419fba-b63b-460b-bff5-d9afa41d0da5)

https://geohack.toolforge.org/geohack.php?pagename=Extremes_on_Earth&params=48_52.6_S_123_23.6_W_type:landmark&title=Point+Nemo
